### ISSUE TITLE
Redirects 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#2296]
The urls listed in this spreadsheet are not redirecting. I've added them to to the file through GitHub in master but the CMS is not showing them as added.  I will look into it , otherwise we'd have to add them manually. ![image.png](https://images.zenhubusercontent.com/5c58acad07cc4d6f39e54f67/1aec4642-1c63-4f9a-b22a-c6fccbf46642)

**Comment by Maroyafaied** [comment]
The missing redirects are in this spreadsheet [Redirects_CLT_Additions_CSV.csv](https://github.com/ONRR/onrr.gov-site/files/10056978/Redirects_CLT_Additions_CSV.csv)

**Comment by Maroyafaied** [comment]
This spreadsheet has all the existing and missing redirects with all the fields required for the CMS. 
[redirects.csv](https://github.com/ONRR/onrr.gov-site/files/10057004/redirects.csv)

**Comment by Maroyafaied** [comment]
@cthomasONRR I added the redirects to dev and main.  The way I did it is that I downloaded the existing file from GitHub>onrr.gov-site>database>static>csv>redirects and I added the 150 missing and filled in the other fields (copy & pasted from previous ones except for the date) then I saved that file and imported it to the CMS.  I reviewed several and they are working except for the unbundling ones. If it's only the few unbundling not working, i'm ok closing this issue.
